docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
 # raiden-services containers
   pfs:
     << : *services-defaults
-    command: ["python3", "-m", "pathfinding_service.cli"]
+    command: ["python3", "-m", "pathfinding_service.cli", "--matrix-server", "https://${SERVER_NAME}"]
```

Comment:
@ulope you're probably a better reviewer for this one

Reply:
I think it makes sense to force the services to the local synapse server.

The only downside is that the services could then not roam to another server if the local one should be down for some reason. But that doesn't seem like a situation worth optimizing for.